build/release.mk

```diff
@@ -97,8 +97,8 @@ package:
 	mkdir -p $(DIST_PATH)/client
 	cp -RL $(BUILD_WEBAPP_DIR)/dist/* $(DIST_PATH)/client
 
-	$(eval MMCTL_REL_TO_DOWNLOAD := $(shell scripts/get_latest_release.sh 'mattermost/mmctl' 'release-'))
-	@echo "Using mmctl version $(MMCTL_REL_TO_DOWNLOAD)"
+  #Download MMCTL
+	$(shell scripts/get_latest_release.sh)
```

Comment:
technically `$(shell ...)` is not needed and `scripts/get_latest_release.sh` can be called directly.